.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,93 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  es-integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Setup go env
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Install tools
+      run: make install-ci
```

Comment:
do we actually need this step for the integration tests?

Reply:
Good point, I looked at travis.yml and it was there in `install` section but I think we mayn't require it. Let me test it once.